### PR TITLE
Refactor sql filters

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Changed

- Rm `FILTER_` from filter object attributes. Harmonize to filter instead of get in filter functions.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b refactor-sql-filters -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/constants/demultiplexing.py [hunk 1]
```diff
@@ -229,7 +229,7 @@ class IndexOverrideCycles(StrEnum):
 BCL2FASTQ_METRICS_DIRECTORY_NAME: str = "Stats"
 BCL2FASTQ_METRICS_FILE_NAME: str = "Stats.json"
 CUSTOM_INDEX_TAIL = "NNNNNNNNN"
-DRAGEN_PASSED_FILTER_PCT: float = 100.00000
+DRAGEN_PASSED_PCT: float = 100.00000
```
Comment (ChrOertlin) [inline cg/constants/demultiplexing.py:232]: is this wanted ?

File: cg/store/filters/status_case_sample_filters.py [hunk 2]
```diff
@@ -40,5 +40,5 @@ def apply_case_sample_filter(
 class CaseSampleFilter(Enum):
     """Define CaseSample filter functions."""
 
-    GET_SAMPLES_IN_CASE_BY_INTERNAL_ID: Callable = get_samples_in_case_by_internal_id
-    GET_CASES_WITH_SAMPLE_BY_INTERNAL_ID: Callable = get_cases_with_sample_by_internal_id
+    GET_SAMPLES_IN_CASE_BY_INTERNAL_ID: Callable = filter_samples_in_case_by_internal_id
+    GET_CASES_WITH_SAMPLE_BY_INTERNAL_ID: Callable = filter_cases_with_sample_by_internal_id
```
Comment (islean) [inline cg/store/filters/status_case_sample_filters.py:44]: Maybe rename these as well? Most homogeneous would be SAMPLES_IN_CASE_BY_INTERNAL_ID and CASES_WITH_SAMPLE_BY_INTERNAL_ID

**Review by islean — APPROVED** [review]
Looks great!

File: cg/constants/demultiplexing.py (hunk 1 above)
Comment (islean) [reply cg/constants/demultiplexing.py:232]: Second this question

Comment (henrikstranneheim) [reply cg/constants/demultiplexing.py:232]: Good catch! Will revert!

File: cg/store/filters/status_case_sample_filters.py (hunk 2 above)
Comment (henrikstranneheim) [reply cg/store/filters/status_case_sample_filters.py]: Fixed

File: cg/constants/demultiplexing.py (hunk 1 above)
Comment (henrikstranneheim) [reply cg/constants/demultiplexing.py]: Done

**Comment by henrikstranneheim** [comment]
cg workflow mip-dna start justhusky -d 👍 
```
 cg workflow mip-dna start justhusky -d
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Case justhusky exists in Status DB
Starting full MIP analysis workflow for case justhusky
Flow cell check is not applicable - the case is either down sampled or external.
Found file earlycasualcaiman/2019-05-08/HHT5NDSXX_HapMap_S26_L001_R1_001.fastq.gz
Found file earlycasualcaiman/2019-05-08/HHT5NDSXX_HapMap_S26_L001_R2_001.fastq.gz
Found file slowlycivilbuck/2019-05-08/HHT5NDSXX_HapMap_S27_L001_R1_001.fastq.gz
Found file slowlycivilbuck/2019-05-08/HHT5NDSXX_HapMap_S27_L001_R2_001.fastq.gz
Found file hugelymodelbat/2019-05-08/HHT5NDSXX_HapMap_S28_L001_R1_001.fastq.gz
Found file hugelymodelbat/2019-05-08/HHT5NDSXX_HapMap_S28_L001_R2_001.fastq.gz
Case justhusky exists in Status DB
Case justhusky exists in Status DB
Running command /home/proj/stage/bin/miniconda3/envs/S_scout/bin/scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml export panel --bed BRAIN Cardiology CILM CH CTD DIAB ENDO EP HEARING HYDRO IBMFS IEM IF mcarta MHT MIT MOVE mtDNA NBS-M NEURODEG NMD OMIM-AUTO OPTIC PANELAPP-GREEN PEDHEP PID PIDCAD RETINA SKD SOVM STROKE AID Inherited cancer --build 37
MT	15956	16023	7494	MT-TP
Case justhusky exists in Status DB
Running command /home/proj/stage/bin/miniconda3/envs/S_scout/bin/scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml export managed --build 37
##fileformat=VCFv4.2
##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
##fileDate=2024-02-27 16:23:31.384523
##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
##INFO=<ID=TYPE,Number=1,Type=String,Description="Type of variant">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	48696925	.	G	C	.		END=48696925;TYPE=SNV
7	124491972	.	C	A	.		END=124491972;TYPE=SNV
12	103234252	.	T	C	.		END=103234252;TYPE=SNV
14	76548781	.	CTGGACC	G	.		END=76548781;TYPE=INDEL
17	48696925	.	G	T	.		END=48696925;TYPE=SNV
Case justhusky exists in Status DB
{'case': 'justhusky', 'default_gene_panels': ['OMIM-AUTO'], 'samples': [{'sample_id': 'earlycasualcaiman', 'sample_display_name': 'ACC5339A2--300M', 'analysis_type': 'wgs', 'sex': 'male', 'phenotype': 'unaffected', 'expected_coverage': 1, 'capture_kit': 'twistexomerefseq_9.1_hg19_design.bed', 'mother': '0', 'father': '0'}, {'sample_id': 'slowlycivilbuck', 'sample_display_name': 'ACC5339A3--300M', 'analysis_type': 'wgs', 'sex': 'female', 'phenotype': 'unaffected', 'expected_coverage': 1, 'capture_kit': 'twistexomerefseq_9.1_hg19_design.bed', 'mother': '0', 'father': '0'}, {'sample_id': 'hugelymodelbat', 'sample_display_name': 'ACC5339A4--300M', 'analysis_type': 'wgs', 'sex': 'male', 'phenotype': 'affected', 'expected_coverage': 1, 'capture_kit': 'twistexomerefseq_9.1_hg19_design.bed', 'mother': 'slowlycivilbuck', 'father': 'earlycasualcaiman'}]}
Case justhusky exists in Status DB
earlycasualcaiman: downsampled/external - skip evaluation
Running command /home/proj/stage/bin/miniconda3/bin/conda run --name S_mip12.0 /home/proj/stage/bin/miniconda3/envs/S_mip12.0/bin/mip analyse rd_dna --config /home/proj/stage/servers/config/hasta.scilifelab.se/mip12.0-dna-stage.yaml justhusky --slurm_quality_of_service high --email henrik.stranneheim@scilifelab.se --qccollect_skip_evaluation
Dry run: process call will not be executed!!

**Comment by henrikstranneheim** [comment]
Deployed to production:

```
Log deploy... done.
cg, version 59.5.6

**Pull request merged**